cerebro/tests/test.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+
+TESTDIR="$(dirname "${0}")"
+PLANDIR="$(dirname "${TESTDIR}")"
+SKIPBUILD=${SKIPBUILD:-0}
+
+hab pkg install --binlink core/bats
+
+source "${PLANDIR}/plan.sh"
+
+if [ "${SKIPBUILD}" -eq 0 ]; then
+  set -e
+  pushd "${PLANDIR}" > /dev/null
+  build
+  source results/last_build.env
+  hab pkg install --binlink --force "results/${pkg_artifact}"
+  hab pkg install --binlink --force "core/curl"
```

Comment:
`core/curl` should be installed outside the `SKIPBUILD` check. Ideally just below/around the `core/bats` install line.

Reply:
Additionally, I don't think it needs to be forced.